EventFilter/HcalRawToDigi/interface/HcalLaserEventFilter2012.h

```diff
@@ -42,27 +40,29 @@ class HcalLaserEventFilter2012 : public edm::EDFilter {
 public:
   explicit HcalLaserEventFilter2012(const edm::ParameterSet&);
   ~HcalLaserEventFilter2012() override;
-  
+
   static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
-  
+
   friend HcalLaserEventFiltProducer2012;
 
 private:
   bool filter(edm::Event&, const edm::EventSetup&) override;
   void endJob() override;
-  
-  void readEventListFile(const std::string & eventFileName);
-  void addEventString(const std::string & eventString);
-
- // ----------member data ---------------------------
-  typedef std::vector< std::string > strVec;
-  typedef std::vector< std::string >::iterator strVecI;
-  
-  std::vector< std::string > EventList_;  // vector of strings representing bad events, with each string in "run:LS:event" format
-  bool verbose_;  // if set to true, then the run:LS:event for any event failing the cut will be printed out
-  std::string prefix_;  // prefix will be printed before any event if verbose mode is true, in order to make searching for events easier
-  
-  // Set run range of events in the BAD LASER LIST.  
+
+  void readEventListFile(const std::string& eventFileName);
+  void addEventString(const std::string& eventString);
+
+  // ----------member data ---------------------------
+  typedef std::vector<std::string> strVec;
+  typedef std::vector<std::string>::iterator strVecI;
+
+  std::vector<std::string>
```

Comment:
Comments could be reshuffled to avoid this line splitting

Reply:
done